mewarpx/mewarpx/assemblies.py

```diff
@@ -316,6 +316,95 @@ def __init__(self, V, T, WF, read_scraped_particles=True):
         self.scraper_label = 'z_lo'
 
 
+class PatchyCathode(object):
+    """A basic wrapper to define a semi-infinite plane for a patchy cathode.
+    Special aspects of the patchy cathode involve instantiation of two assembly
+    objects for the low and high work-function patches of the cathode and
+    setting a spatially varying boundary potential. The spatially varying
+    boundary potential is simplified by using an embedded boundary rather than
+    the domain boundary. Simplified assemblies similar to ``Rectangle`` is used.
```

Comment:
Maybe clarify that the "simplification" is for the specific installation of assemblies and potentials in WarpX/mewarpx. It's not conceptually simpler!